.konflux/scripts/subscription-manager-bro.sh

```diff
@@ -12,8 +12,9 @@ SCRIPT_DIR="$( cd -- "$( dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)"
 
 SECRET_NAME_IN_KONFLUX="subscription-manager-activation-key"
 SECRET_KONFLUX_WORKSPACE_PATH="/workspace/${SECRET_NAME_IN_KONFLUX}"
+SECRET_MOUNT_PATH="/activation-key"
 SECRET_KEY="activation-key"
-SECRET_LOCAL_PATH="${SCRIPT_DIR}/${SECRET_KEY}"
+SECRET_LOCAL_PATH="${SECRET_MOUNT_PATH}/${SECRET_KEY}"
```

Comment:
This unfortunately breaks `self-test` (for which you need to create `activation-key` file locally and copy `.konflux/scripts/subscription-manager/bro.self-test.*` files).

Since we likely want to use the new approach in the stackrox as well and still be able to self-test the approach, could you please find a way to make it pass self-test?

Reply:
I adapted the self-tests in https://github.com/stackrox/stackrox/pull/12147.

Reply:
https://github.com/stackrox/stackrox/pull/12147 is merged and https://github.com/stackrox/collector/pull/1757/commits/2b1335d4bf02a07e5d72a28e0eb0847323c8adc4 updates the scripts in this PR.